pkgs/applications/editors/emacs/default.nix

```diff
@@ -32,20 +32,6 @@ lib.makeScope pkgs.newScope (self:
         fetchFromSavannah;
     };
 
-    emacs28 = callPackage (self.sources.emacs28) inheritedArgs;
```

Comment:
You are not merely setting 29 as default, you are effectively removing 28.

Put this on the description too.